### PR TITLE
test: api_blkio_test: add new checkpoints

### PR DESCRIPTION
**Pull request by chuanchang**
Add new checkpoints into blkio testing
Signed-off-by: Alex Jia <chuanchang.jia@gmail.com>

### Ⅰ. Describe what this PR did
As summary.

### CODE REVIEW

File: apis/opts/config/blkio_test.go [hunk 1]
```diff
@@ -209,7 +217,19 @@ func TestWeightDevice_Type(t *testing.T) {
 		fields fields
 		want   string
 	}{
-	// TODO: Add test cases.
+		// TODO: Add test cases.
```
Comment (fuweid) [inline apis/opts/config/blkio_test.go]: could we remove this useless line?

File: apis/opts/config/blkio_test.go [hunk 2]
```diff
@@ -455,7 +520,32 @@ func TestThrottleBpsDevice_value(t *testing.T) {
 		fields fields
 		want   []*types.ThrottleDevice
 	}{
-	// TODO: Add test cases.
+		// TODO: Add test cases.
+		{
```
[the remark this answers is not in the document]
Comment (chuanchang) [reply apis/opts/config/blkio_test.go]: @Ace-Tang IMO, it's not necessary to add invalid string for Path and invalid value for Rate check in here, in other words, it's enough to cover current code path.

**Comment by chuanchang** [comment]
@fuweid useless lines have been removed from v2 patch

**Comment by allencloud** [comment]
LGTM

File: apis/opts/config/blkio_test.go [hunk 3]
```diff
@@ -209,7 +217,18 @@ func TestWeightDevice_Type(t *testing.T) {
...
+			name: "implement WeightDevice as pflag.Value interface",
+			fields: fields{
+				values: []*types.WeightDevice{
+					{
+						Path:   "device1",
+						Weight: uint16(1000),
+					},
+				},
+			},
+			want: "value",
```
Comment (HusterWan) [inline apis/opts/config/blkio_test.go]: Why "value" not "strings" ? @chuanchang

**Pull request merged**